<launch>

	<node pkg="gps_common" type="utm_odometry_node" name="utm_odometry_node"  output="screen" > 
	  
	  <param name="rot_covariance"  type="double" value="99999" />
	  <param name="frame_id"        type="string" value="imu" />
	  <param name="child_frame_id"  type="string" value="odom" />

		<remap from="fix"       to="/vectornav/ins/NavSatFix" />
	</node>
	
		<!-- publish static transform -->
	<node pkg="tf" type="static_transform_publisher" name="vectornav_tf_odom" 
	      args="0 0 0 0 0 0 map imu 100" />
	      
	<node pkg="vectornav" type="local_odom" name="local_odom" />
</launch>


